src/bll/processReducer.ts

```diff
@@ -1,14 +1,19 @@
-import {createSlice} from '@reduxjs/toolkit';
-import {quizIdMoc} from '@src/Mocs/Testing';
-
-const initialState = quizIdMoc;
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {quizTypeResponse} from '@customTypes/quiz-types';
 
+const initialState = {
+  quiz: {} as quizTypeResponse,
```

Comment:
if you have only one key, you can do like: `const initialState = {} as quizTypeResponse `